apps/remote_control/lib/lexical/remote_control/api.ex

```diff
@@ -47,6 +48,10 @@ defmodule Lexical.RemoteControl.Api do
     ])
   end
 
+  def struct_doc(%Project{} = project, struct_module) do
```

Comment:
I think the API here could be designed to be more versatile, not just limited to struct doc.